packages/ast-tooling/index.ts

```diff
@@ -173,9 +173,9 @@ export function parseJson(content: string) {
     return fleece.evaluate(content);
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function serializeJson(originalInput: string, data: any) {
+export function serializeJson(originalInput: string, data: unknown) {
     // some of the files we need to process contain comments. The default
     // node JSON.parse fails parsing those comments.
-    return fleece.patch(originalInput, data);
+    // return fleece.patch(originalInput, data);
+    return fleece.stringify(data);
```

Comment:
From the docs for [`silver-fleece`](https://github.com/evertheylen/silver-fleece/blob/master/README.md), it looks like `patch` is used to preserve the formatting, but it's also interfering with our attempts of alphabetizing the dependencies. 

For instance, when we use the tailwind adder with the typography plugin included, all of the newly added deps are still be appended to the end of the dependency map:
![img](https://i.imgur.com/uIZWA6H.png)